src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,9 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Limits file saving to only command types that mutate data
```

Comment:
Great that you've update the comment header for the method. However, It is better to mention it returns true if the Command potentially mutates the data (header comments specify a contract, so try to be as precise as possible)